src/core/network.js

```diff
@@ -75,6 +75,21 @@ var NetworkManager = (function NetworkManagerClosure() {
     return array.buffer;
   }
 
+//#if (GENERIC || FIREFOX || B2G || MOZCENTRAL)
```

Comment:
Some additional information:
It seems that `moz-chunked-arraybuffer` was added in https://bugzilla.mozilla.org/show_bug.cgi?id=687087, which landed in Firefox 9 (released on 2011-12-20). I would be *very* surprised if the addon even works in such an old version of Firefox anyway, so it *should* be fine.